app/controllers/application_controller.rb

```diff
@@ -53,7 +66,7 @@ def authorize_resource(resource, action)
 
   def check_no_institution!
     return if current_user && current_user.need_change_password?
-    if current_user && current_user.institutions.empty? && current_user.policies.empty?
+    if current_user && current_user.institutions.empty? && current_user.policies.empty? && !is_super_admin?
```

Comment:
This check is redundant: In order to be a "super admin" the user needs to have a policy, so the previous check would fail and this last part will never be executed.